packages/events/src/service/indexing/indexing.ts

```diff
@@ -169,6 +169,16 @@ function mapFieldTypeToElasticsearch(field: FieldConfig) {
           type: { type: 'keyword' }
         }
       }
+    case FieldType.FILE_WITH_OPTIONS:
+      return {
+        type: 'nested',
```

Comment:
```suggestion
        type: 'object',
```

Or was there a reason you'd go with nested instead?

Reply:
This is an array of objects.

Reply:
Ok I see 👍